package-lock.json

```diff
@@ -1,10248 +0,0 @@
-{
```

Comment:
Please don't delete this file. Simply revert the changes.

Reply:
ok sir will do it